packages/netlify-cms-core/src/reducers/editorialWorkflow.ts

```diff
@@ -121,14 +132,23 @@ const unpublishedEntries = (state = Map(), action: EditorialWorkflowAction) => {
       );
 
     case UNPUBLISHED_ENTRY_PUBLISH_SUCCESS:
+      return state.deleteIn(['entities', `${action.payload!.collection}.${action.payload!.slug}`]);
```

Comment:
Can we merge the `UNPUBLISHED_ENTRY_PUBLISH_SUCCESS` and `UNPUBLISHED_ENTRY_PUBLISH_FAILURE` cases?

Reply:
ohh you're right 😃 my bad